src/test/java/no/unit/nva/model/instancetypes/journal/JournalArticleTest.java

```diff
@@ -18,14 +27,79 @@ void journalArticleReturnsObjectWhenPeerReviewIsTrueOrFalse(boolean peerReviewed
         assertDoesNotThrow(() -> generateJournalArticle(testData));
     }
 
+    @Test
```

Comment:
We also need a test that we test that the serialization form is `"content" : <string>`

Reply:
Made a simple one..